pennylane/gradients/parameter_shift_hessian.py

```diff
@@ -67,10 +67,10 @@ def _process_jacs(jac, qhess):
         qh_indices = "ab..."
 
         # contract the first axis of the jacobian with the first and second axes of the Hessian
-        first_jac_indices = f"a{ABC[2:2 + jac_ndim - 1]}"
-        second_jac_indices = f"b{ABC[2 + jac_ndim - 1:2 + 2 * jac_ndim - 2]}"
+        first_jac_indices = f"a{ascii_letters[2:2 + jac_ndim - 1]}"
+        second_jac_indices = f"b{ascii_letters[2 + jac_ndim - 1:2 + 2 * jac_ndim - 2]}"
 
-        result_indices = f"{ABC[2:2 + 2 * jac_ndim - 2]}..."
+        result_indices = f"{ascii_letters[2:2 + 2 * jac_ndim - 2]}..."
```

Comment:
:thinking: 
```suggestion
        result_indices = f"{first_jac_indices}{second_jac_indices}..."
```

Reply:
Are these equivalent? Any concern with the `a` and `b` in both that aren't implicitly in the original `result_indices`?

Reply:
Oh, you are absolutely right, I did overlook those :/ 
May I make another suggestion to improve readability? I.e., discard the one in this thread above, and combine all other suggestions. In my opinion this would make the code much closer to the dev comment just above :)

Reply:
Sure, no reason not to clean stuff up while we're here. I'll accept your other suggestions into the PR. Thanks @dwierichs